doc/release_notes-16-08.txt

```diff
@@ -16,7 +16,7 @@ Interactive and dynamic workloads on top of the seL4 kernel
 ###########################################################
 
 seL4 is a modern microkernel that undergoes verification and promises to be
-a well foundation for trustworthy systems. Genode, as operating system
```

Comment:
I don't understand the ``good`` and ``well`` distinction myself, but this is a case of ``good``